scripts/on_sd_start.sh

```diff
@@ -72,7 +72,7 @@ export SD_UI_PATH=`pwd`/ui
 export ED_BIND_PORT="$( python scripts/get_config.py --default=9000 net listen_port )"
 case "$( python scripts/get_config.py --default=False net listen_to_network )" in
     "True")
-        export ED_BIND_IP=0.0.0.0
+        export ED_BIND_IP=$( python scripts/get_config.py --default=0.0.0.0) net bind_ip)
```

Comment:
`--default=0.0.0.0) net bind_ip)`

Is there an extra parantheses-end after 0.0.0.0?

Reply:
Fixed